gf-webapp-admin/src/component/rooms/roomsList/RoomsList.module.scss

```diff
@@ -1,5 +1,4 @@
 .container {
-  display: grid;
```

Comment:
See the cooment above

Reply:
The main reason why I changed grid to flex was that blocks with room rendering with different size depending on the number of rooms. I think it's a bad idea to back old styles because it's wrong view.